app.json

```diff
@@ -18,7 +18,7 @@
       "url": "https://github.com/heroku/heroku-buildpack-ruby.git"
     },
     {
-      "url": "https://github.com/ably-forks/heroku-buildpack-nanoc"
+      "url": "https://github.com/ably-forks/heroku-buildpack-nanoc.git"
```

Comment:
P.S. We don't normally do merges like this because it results in pretty messy commit history once this PR is merged in.  I can appreciate in this instance you wanted this commit put onto this PR so that it builds, however I would have preferred to see a `cherry-pick` of the commit you need, and when this PR is later approved, when rebased with the `interactive` option, the cherry-picked commit would probably be automatically removed anyway as it would already be in the `master` branch you are rebasing on.